client/src/partials/subsidy/subsidy.js

```diff
@@ -0,0 +1,103 @@
+angular.module('bhima.controllers')
+.controller('subsidy', [
+  '$scope',
+  '$translate',
+  '$http',
+  'validate',
+  'messenger',
+  'connect',
+  'appstate',
+  'uuid',
+  function ($scope, $translate, $http, validate, messenger, connect, appstate, uuid) {
+    var dependencies = {},
+        session = $scope.session = {};
+
+    dependencies.subsidies = {
+      identifier:'uuid',
+      query : '/getSubsidies/'
```

Comment:
This query should be done with the validate syntax.

Additionally, it is bad practice to create case-sensitive URLs, as many webservers do not take into account the string case.  This should be avoided in the future.